<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiMyAvatar</class>
 <widget class="QDialog" name="uiMyAvatar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>379</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>522</width>
    <height>362</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>487</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>My avatar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../scc.qrc">
    <normaloff>:/images/logo16x16.png</normaloff>:/images/logo16x16.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_my_avatar">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_my_nick">
         <property name="styleSheet">
          <string notr="true">font-weight:bold;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="AvatarListWidget" name="avatarListWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>520</width>
         <height>272</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AvatarListWidget</class>
   <extends>QWidget</extends>
   <header>src/widgets/avatar_list_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../scc.qrc"/>
 </resources>
 <connections/>
</ui>
